Sources_v3/StreamChatUI/Utils/UITextView+Extensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+extension UITextView {
```

Comment:
I'm not sure putting this into an extension is a good idea. It doesn't allow anyone to override it and change it.

Reply:
But I don't expect this method to be overriden.
It can be changed in future if there will be some adjustments and they will be applied to all places it is used and that's how I want this to work.

Reply:
Is there any specific reason you to explicitly prevent users from overriding this?

Reply:
Cause the purpose of this property is to return fitting height of `UITextView` and it should work the same way all the time for all `UITextViews`. There is no need to override it cause behaviour should be the same. If someone want to override it then someone needs new different property that is not related to this one.